daprdocs/content/en/operations/troubleshooting/common_issues.md

```diff
@@ -226,3 +226,26 @@ In order for mDNS to function properly, ensure `Micorosft Content Filter` is ina
 Microsoft Content Filter is disabled when the output is "Success".
 
 > Some organizations will re-enable the filter from time to time. If you repeatedly encounter app-id values missing, first check to see if the filter has been re-enabled before doing more extensive troubleshooting.
+
+## I can not create a pod
```

Comment:
Please add a more descriptive title. What pod is not being created? this is too generic